_i18n/en.yml

```diff
@@ -124,14 +124,42 @@ sponsorships:
-  developers_para2: OpenHub contributors page.
-  developers_para3: Below you'll find some individuals that have gone above and beyond for Monero.
-  community: Community
+  introduction: Workgroups are formed by community members who join forces to achieve a common goal. They have different structures and goals. On this page we list some of the more active workgroups and some tips to get you started, if you wish to contribute to any of them.
+  core: Core Team
+  community: Community Workgroup
   mrl: Research Lab
-  thanks: Special Thanks
+  descr: Description
+  start: Where to start
+  contacts: Chatrooms and Contacts
+  chat: "Chat:"
+  website: "Website:"
+  core_descr: The members of this workgroup are always 7. They are the stewards of the community and they manage and maintain the Monero codebase and infrastructure. The Core Team act as mediators, weighting consensus in the community and taking decisions according to community's will.
+  core_start: The Core Team is a closed workgroup, it's not possible to join it unless invited.
+  community_descr: Workgroup used by the community to organize and coordinate. This is the place where every non-coder who wants to contribute to Monero should start.
+  community_start: The best way to start contributing to this workgroup is to join the chatrooms and introduce yourself, stating your interests and the skills you would like to offer to the community.
+  devworkgroup: Development Workgroup
+  devworkgroup_descr: This workgroup reunites Monero developers working on the wallets and daemon maintained by the Core Team. They are mostly divided in GUI and CLI devs and they coordinate using two different chatrooms.
+  devworkgroup_start: It's a good idea to take a look at open issues on both repositories to have an idea of what could be fixed or improved. Both codebases, but especially the core repository, can be quite challenging, so it's suggested to first write in the chatroom introducing yourself, your skills and what you wish to contribute to.
+  chatgeneral: "Chat (general):"
+  chatgui: "Chat (GUI):"
+  webworkgroup: Website Workgroup
+  web_descr: Another dev-focused workgroup where to coordinate the development of the website. Most devs in there are part of the development workgroup. The website workgroup was created mostly to have discussions related to the website all in one place.
+  web_start: It's always a good idea to join the chatroom and talk directly with the members of the workgroup, but if you prefer to get started by yourself, take a look at the readme in the GitHub repository. It contains step by step instructions to edit every single part of this website.
+  localization: Localization Workgroup
+  localization_descr: One of the oldest Monero Workgroups. It's formed by contributors working on translating Monero-related software and documentation. It's strongly suggested to contact the workgroup before working on translations.
```

Comment:
Recommend removing first sentence, then changing the beginning of the second sentence to "This workgroup consists of contributors who translate..."

Reply:
Why removing the first sentence? it's an anecdotal reference, but at least makes the "description" field less monotone

Reply:
Just because it doesn't describe anything relevant/actionable to the audience imo.

Reply:
I disagree. The localization workgroup is the oldest organized workgroup of the community. IMO it's relevant to mention it in a page dedicated to Monero workgroups and it's also something that makes me proud. It's a description, doesn't need to be actionable.